### PR TITLE
Aktivnost za dan 2024-05-10 - 2021240408

### PR DESCRIPTION
**Pull request by jovansimonovic**

### CODE REVIEW

**Review by pbisevac — CHANGES_REQUESTED** [review]
Can you add some validation for User Model? @jovansimonovic

File: src/main/java/com/example/demo/models/UserModel.java [hunk 1]
```diff
@@ -7,5 +7,7 @@ public class UserModel {
   private int id;
   private String firstName;
   private String lastName;
+  private String email;
```
Comment (pbisevac) [inline src/main/java/com/example/demo/models/UserModel.java:10]: Add some validation od fields

**Review by pbisevac — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/demo/models/UserModel.java (hunk 1 above)
Comment (jovansimonovic) [reply src/main/java/com/example/demo/models/UserModel.java:14]: Added validation for firstName, lastName and email. Added failure message for invalid user registration attempts (server returns "Neuspesno registrovan")

**Review by pbisevac — APPROVED** [review]
Well done, you can merge

**Pull request merged**